src/components/Bar.jsx

```diff
@@ -93,11 +93,11 @@ class Bar extends Component {
   }
 
   renderLeft () {
-    const { t } = this.props
+    const { t, displayOnMobile, isPublic } = this.props
     // data-tutorial attribute allows to be targeted in an application tutorial
-    return <button className='coz-bar-burger' onClick={this.toggleDrawer} data-icon='icon-apps' data-tutorial='apps-mobile'>
+    return (__TARGET__ !== 'mobile' || displayOnMobile) && !isPublic ? <button className='coz-bar-burger' onClick={this.toggleDrawer} data-icon='icon-apps' data-tutorial='apps-mobile'>
```

Comment:
Why is `public` mentioned ?

Reply:
This is the exact condition that was previously used. IIRC, we also hide this button if we're on a public page, ie. the public page for a shared album.